whyis/blueprint/entity/get_entity.py

```diff
@@ -0,0 +1,50 @@
+from flask import current_app, request, Response, make_response
+from rdflib import ConjunctiveGraph
+from werkzeug.exceptions import abort
+from depot.middleware import FileServeApp
+
+from .entity_blueprint import entity_blueprint
+from whyis.data_extensions import DATA_EXTENSIONS
+from whyis.data_formats import DATA_FORMATS
+from whyis.decorator import conditional_login_required
+
+import sadi.mimeparse
+
+from whyis.html_mime_types import HTML_MIME_TYPES
+
+@entity_blueprint.route('/about.<format>', methods=['GET'])
+@entity_blueprint.route('/<path:name>', methods=['GET'])
+@entity_blueprint.route('/<path:name>.<format>', methods=['GET'])
+@entity_blueprint.route('/', methods=['GET'])
+@entity_blueprint.route('/home', methods=['GET'])
+@entity_blueprint.route('/about', methods=['GET'])
+@conditional_login_required
+def view(name=None, format=None, view=None):
+    current_app.db.store.nsBindings = {}
+    entity, content_type = current_app.get_entity_uri(name, format)
+    
+    resource = current_app.get_resource(entity)
+
+    # 'view' is the default view
+    fileid = resource.value(current_app.NS.whyis.hasFileID)
+    if fileid is not None and 'view' not in request.args:
+        print (resource.identifier, fileid)
+        f = current_app.file_depot.get(fileid)
+        fsa = FileServeApp(f, current_app.config["file_archive"].get("cache_max_age",3600*24*7))
+        return fsa
+            
+    if content_type is None:
+        content_type = request.headers['Accept'] if 'Accept' in request.headers else 'text/turtle'
```

Comment:
Should probably log a warning on missing Accept before falling back, Owen and I got tripped up on this before.